packages/calcite-components/src/components/menu-item/menu-item.tsx

```diff
@@ -437,13 +437,26 @@ export class CalciteMenuItem implements LoadableComponent, T9nComponent, Localiz
     );
   }
 
+  renderHrefIcon(dir: Direction): VNode {
+    return (
+      <calcite-icon
+        class={CSS.hoverHrefIcon}
+        icon={dir === "rtl" ? "arrow-left" : "arrow-right"}
+        key={CSS.dropdownAction}
```

Comment:
Can `key` be `"hrefIcon"` or something along those lines? I find it hard to understand the association between the dropdown action CSS class and the href icon.

Reply:
Wrong class used, fixed.